pkg/config/config_unsupported.go

```diff
@@ -26,3 +26,7 @@ func selinuxEnabled() bool {
 func checkKernelRROMountSupport() error {
 	return errdefs.ErrNotImplemented
 }
+
+func (c *RuntimeConfig) ValidatePinnsPath(executable string) error {
```

Comment:
should we add a gh action that compiles on mac so you don't need to keep playing catchup each time you commit to cri-o?

Reply:
I’d _personally_ like that, but if you don’t have it already, you probably don’t need it to be effective in maintaining the software.

(There’s _much_ more in CRI-O that doesn’t compile on Mac; this was only targeted at making `internal/storage` unit tests run.)